scripts/check_containers.sh

```diff
@@ -1,11 +1,14 @@
 #!/usr/bin/env bash
-# Check if there were any dockerfiles modified in the latest pull request.
-# Note: this is specific to the way Azure checks out git repo.
+# Check if there were any dockerfiles modified in the PR.
+# Note: this is specific to Azure - in the pipeline's local branch git history
+# contains one merge commit for all of the changes in the PR.
 
 diff_latest() {
-  git diff --name-only "$(git log --skip=1 -1 --merges --pretty=format:%H)"..
+  printf "Files modified in the pull request:\n" >&2
+  git diff --name-only HEAD~ HEAD | tee >(cat >&2)
```

Comment:
```suggestion
  git diff --name-only HEAD^1 HEAD | tee >(cat >&2)
```
Maybe that would work better, if HEAD is a merge commit. It will give the first parent (i.e. the tip of the target branch), rather than a unique parent.

Reply:
Yeah, I was scratching my head thinking what could be the issue. I will give it a try :+1: